go/src/koding/klient/machine/index/change.go

```diff
@@ -180,6 +199,12 @@ func (c *Change) Coalesce(newer *Change) *Change {
 	}
 }
 
+// String implements fmt.Stringer interface. It pretty prints stored change.
+func (c *Change) String() string {
+	age := time.Now().UTC().Sub(time.Unix(0, atomic.LoadInt64(&c.createdAt)))
```

Comment:
Because, there is [logic](https://github.com/koding/koding/blob/10b9a090e637a38cb9ba526df4862bf0f34cc910/go/src/koding/klient/machine/index/change.go#L196) which may replace this field.

It is used by anteroom [logic](https://github.com/koding/koding/blob/3d9e5d90ad60a0a80566ef4fa4a36098dc984430/go/src/koding/klient/machine/mount/sync/anteroom.go#L82)(see comment) which may invalidate change that has been already sent to execution(sync) [workers](https://github.com/koding/koding/blob/3d9e5d90ad60a0a80566ef4fa4a36098dc984430/go/src/koding/klient/machine/machinegroup/syncs/syncs.go#L126).